src/System.Net.Http/src/System/Net/Http/HttpUtilities.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Net.Http
     internal static class HttpUtilities
     {
         internal static Version DefaultRequestVersion =>
-#if uap
+#if !net46
```

Comment:
We don't build a 'net46' target anymore.  That's only done now in the release/2.0.0 branch. So, the if/end-if can probably go away.

Reply:
That said, you'll may have to modify some tests because NETFX won't like getting 2.0 as the version number in the HttpRequestMessage.  It will throw exceptions.

Reply:
Ok, I will test on my dev box first.